mix.exs

```diff
@@ -52,6 +52,8 @@ defmodule Membrane.RTP.AAC.MixProject do
     [
       {:membrane_core, "~> 0.5.1"},
       {:membrane_aac_format, "~> 0.1.0"},
+      {:membrane_remote_stream_format, "~> 0.1.0"},
+      {:membrane_rtp_format, github: "membraneframework/membrane_rtp_format", branch: :develop},
```

Comment:
What do you think about merging and releasing at first membraneframework/membrane_rtp_format#4? We would have less tricky dependencies on master branches.